apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/service/OnboardingServiceDefault.java

```diff
@@ -140,6 +143,31 @@ public Uni<Onboarding> persistAndStartOrchestrationOnboarding(Onboarding onboard
         }
     }
 
+    private WorkflowType getWorkflowType(Onboarding onboarding) {
+        InstitutionType institutionType = onboarding.getInstitution().getInstitutionType();
+        if(InstitutionType.PT.equals(institutionType)){
```

Comment:
is this workflow document somewhere? I'd add link in case to the docs or a big comment here to explain these conditions